application/src/App.tsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import './App.css';
-import { NavBar } from './navigation/NavBar/NavBar';
-import { PageRoutes } from './navigation/PageRoutes/PageRoutes';
+import { NavBar } from './pages/NavBar/NavBar';
+import { PageRoutes } from './pages/PageRoutes/PageRoutes';
 
 function App() {
   return (
-    <div>
+    <div className='.app'>
```

Comment:
1. I would recommend to use such format of attributes declaration
`<div className="app"` - double quotes

2. it's incorrect to put ".app" into className - you have to pass only "app"